lib/generators/matestack_component/templates/matestack_component.haml.erb

```diff
@@ -0,0 +1,5 @@
+.<%= file_name %>{@tag_attributes}
+  %p
```

Comment:
I wouldn't use `p` tag as an example here. It is not valid to yield tags like `div` inside a `p` tag. I actually didn't know that and learned it the hard way. If you provide this example, it might be easy for users to make this mistake (https://stackoverflow.com/questions/10763780/putting-div-inside-p-is-adding-an-extra-p)

Reply:
Happy to swap it for an `div` tag then!